templates/blog/post_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Post's List:</h1>
+  <ul>
+    {% for post in post_list%}
+      <li><a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a>
```

Comment:
The URL tag seems to be using 'post-detail'. Ensure that the URL name matches the one defined in your 'urls.py'. If it's 'post_detail', make sure to use underscores consistently.